src/yaml/ulissewolf/Mediterraneo/ulissewolf-mediterranean-transport.set-vol-1.yaml

```diff
@@ -0,0 +1,27 @@
+group: "ulissewolf"
+name: "mediterranean-transport-set-vol-1"
+version: "1.0"
+subfolder: "700-transit"
+assets:
+- assetId: "ulissewolf-mediterranean-transport-set-vol-1"
+dependencies:
+- "network-addon-mod"
+- "bsc:mega-props-mbear-vol01"
+- "bsc:mega-props-mbear-vol02"
+- "bsc:mega-props-sg-vol01"
+- "bsc:mega-props-dae-vol01"
+- "bsc:textures-vol01"
+- "bsc:textures-vol03"
+info:
+  summary: "This lot was developed to expand the set of stations present in the Mediterranean set made by MBEAR"
```

Comment:
The summary should be short and concise, e.g.:

`42 Mediterranean Hybrid-Railway stations`